app/email-incoming/circulars/index.ts

```diff
@@ -201,7 +201,7 @@ If you have not already done so, we encourage you to make an account at ${origin
 - Your Circulars settings have been transferred automatically.
 - You are able to submit Circulars from the same email addresses registered in the legacy service.
 - Emails from GCN come from a new address, no-reply@${hostname}.
-- We encourage you to submit Circulars to the new address, circulars@${hostname}, but we still support the old address gcncirc@capella2.gsfc.nasa.gov.
+- We encourage you to submit Circulars to the new address, circulars@${hostname}. The old GCN Circulars submission address, gcncirc@capella2.gsfc.nasa.gov, will be shut down and retired on December 31, 2024.
```

Comment:
Doesn't "shut down" and "retired" imply the same thing?

Reply:
Sure. Which term do you prefer?

Reply:
```suggestion
- We encourage you to submit Circulars to the new address, circulars@${hostname}. The old GCN Circulars submission address, gcncirc@capella2.gsfc.nasa.gov, will be retired on December 31, 2024.
```